airbyte-integrations/connectors/source-github/source_github/schemas/deployments.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
This part is not really need
` "$schema": "http://json-schema.org/draft-07/schema#"`